### ISSUE TITLE
Character Stat Tooltip Improvements

### ISSUE THREAD

**Issue by StardustD2** [DestinyItemManager/DIM#9866]
# Sort of a feature request, sort a report for weird formatting

v1.8 brought some changes and improvements on the Clarity side of things and some of those are perhaps worth migrating over to DIM's implementation as well.

### Here's a list of changes that could be made to improve the feature:
- Replace/expand upon the stat descriptions provided in the API with the fully localized ones in the Clarity repo. 
  - These get around the severely outdated ones in the API ("Stasis abilities have longer cooldowns than Arc, Solar, Strand, and Void abilities.") and add some relevant insights about the terminology used and about the behavior of abilities under that stat.
  - Could be tied to the "Community Perk Insights" toggle in settings
- Potentially add a hover tooltip or something similar to the sub-stats like 'Walk Speed' with Mobility
  - These also have localized versions in the Clarity repo and provide additional context alongside just the name of the sub-stat
  - Related: It's a bit unreliable to mouse over to the popup which leads to it going away when you want to mouse over the text there. This would probably need addressing if hover tooltips were to happen. (This also affects the links present in certain perk tooltips that are often vital to explain a mechanic. This means you can almost never click on the link unless you switch to displaying to weapon's perks in list mode.)
    - Perhaps possible to allow clicking on the perk/character stat to keep the popup open until you click away?
- Recovery stat info improvements:
  - "Time To Full HP" was a term I came up with and it is exceedingly unhelpful in conveying its meaning so it would be nice to replace with with the new "Total Regen(eration) Time"
  - Include the 4 new sub-stats that explain the more granular mechanics of health regeneration
  - Change the time format for regeneration from `[mm]:[ss]` to just `[ss]` (the left digit would actually still remain unused) 
    - This would need to have a unit attached like `m/s` with Mobility's movement speeds
- Completely optional Intellect changes:
  - The repository now includes the Damage Resistance values of Supers alongside their conditions so that could potentially be implemented. However, that's completely optional as it'll also be covered in the Community Insights directly on the ability itself once I get around to adding those. It just might be convenient to have them there potentially, idk. It's mostly an experiment to see if that sticks.

**Comment by bhollis** [comment]
Yeah, our tooltips are not designed to allow interacting with their content.

**Comment by StardustD2** [comment]
Figured as much. Though I'd still say that the ability to click/tap them to pull up the tooltip permanently until you click away would still be very useful even without support for additional interactions.
Been using DIM on my phone quite a bit lately and every tooltip tied to icons near the top of the screen gets really awkward to use when I have to rotate my fingers around to read what's there.
Maybe I should make a separate feature request issue with this?

**Comment by bhollis** [comment]
Yeah probably not directly on topic for this issue. We do capture the pointer though, so on touch devices as long as you keep your finger down you can move it anywhere on the screen once the presstip has appeared, for easy reading.
